docs/ref-info/glossary-uid.md

```diff
@@ -460,6 +460,7 @@ import Link from '@docusaurus/Link';
 <dt><MdxJumpAnchor id="gl-uid2-token"><a href="#gl-uid2-token">UID2 Token (Advertising Token)</a></MdxJumpAnchor></dt>
 <dd>A Unified ID 2.0 (UID2) token, also called an advertising token, is an encrypted form of a <a href="#gl-raw-uid2">raw UID2</a>.</dd>
 <dd>UID2 tokens are generated from hashed or unhashed email addresses or phone numbers that are converted to raw UID2s and then encrypted. The UID2 token is a unique value; no two UID2 tokens are the same. UID2 tokens are case sensitive.</dd>
+<dd>The token value is opaque: No assumptions can be made about the format or about the length of the string.</dd>
```

Comment:
`should` be made?
You "can" make false assumptions :)

Reply:
Oops! I saw you'd approved, so went ahead and merged, then saw this. Fixing right now.